src/libclipper/include/clipper/query_processor.hpp

```diff
@@ -6,6 +6,7 @@
 #include <tuple>
 #include <utility>
```

Comment:
Add `#include <stdexcept>` (it's better to explicitly include all the headers needed)